deploy_sns.sh

```diff
@@ -22,15 +22,21 @@ ic-admin  \
    --proposal-title "Let me SNS!"  \
    --summary "This proposal whitelists developer's principal to deploy SNS"
 
-dfx sns import
+${DFX} nns import --network-mapping "${DX_NETWORK}=mainnet"
+${DFX} sns import
 if [ "${CANISTER_TEST}" == "_test" ]
 then
   curl -L "https://raw.githubusercontent.com/dfinity/ic/${IC_COMMIT}/rs/nns/governance/canister/governance_test.did" -o ./candid/nns-governance.did
   curl -L "https://raw.githubusercontent.com/dfinity/ic/${IC_COMMIT}/rs/sns/governance/canister/governance_test.did" -o ./candid/sns_governance.did
 fi
+curl -L "https://github.com/dfinity/nns-dapp/blob/${IC_COMMIT}/sns_aggregator/sns_aggregator.did" -o ./candid/sns_aggregator.did
+cat <<< $(jq -r 'del(.canisters."internet_identity".remote)' dfx.json) > dfx.json
+cat <<< $(jq -r 'del(.canisters."nns-dapp".remote)' dfx.json) > dfx.json
+cat <<< $(jq -r 'del(.canisters."sns_aggregator".remote)' dfx.json) > dfx.json
+
```

Comment:
Why didn't we need the sns_aggregator before?

Reply:
Currently, we're using the dfx.json files in the cloned nns-dapp and internet-identity repositories:
- an advantage is that we don't need to touch the dfx.json file in the repository's root (e.g., remove the "remote" section added by `dfx nns import` assuming the canisters have been deployed during `dfx nns install`; in our case we deploy pinned versions of II, nns-dapp, and aggregator separately);
- a disadvantage is that we need to clone those huge repositories only for the sake of their dfx.json files (which can also break our setup if, e.g., they specify a dfx version that we don't yet support in this repository).